util/MsSqlMigratorUtility/Program.cs

```diff
@@ -32,10 +32,11 @@ private static void WriteUsageToConsole()
 
     private static bool MigrateDatabase(string databaseConnectionString, bool verbose = false, bool repeatable = false, string folderName = "")
     {
-        Console.WriteLine($"repeatable: {repeatable}");
-        Console.WriteLine($"folderName: {folderName}");
         var logger = CreateLogger(verbose);
 
+        logger.LogInformation($"repeatable: {repeatable}");
```

Comment:
🎨 Since we're here let's emit a readable log e.g.:

`logger.LogInformation($"Migrating database with repeatable: {repeatable} and folderName: {folderName}.");`

Reply:
Done